torch/_refs/__init__.py

```diff
@@ -1471,6 +1471,26 @@ def logaddexp(a: TensorLikeType, b: TensorLikeType) -> TensorLikeType:
         return torch.where(inf_mask, a, max_ + torch.log1p(torch.exp(min_ - max_)))
 
 
+@_make_elementwise_binary_reference(
```

Comment:
Side note - does this automatically register the operator as a decomposition as well?

Reply:
Yes it gets registered by this line,
https://github.com/pytorch/pytorch/blob/def1b57151687abd585e3000dd10907b8be01266/torch/_refs/__init__.py#L938